src/static/css/module/brand-header.less

```diff
@@ -0,0 +1,32 @@
+.brand-header {
+  background-color: @gray-10;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  border-bottom: 1px solid @gray-50;
+
+  &__icon {
+    color: @green;
+    font-size: 2.5em;
+    text-align: center;
+    .grid_column(2);
```

Comment:
I refactored a few things and I think it works with absolute positioning. I wanted to avoid setting the icon in the CSS to make it easier for other tools to configure. Check it out in 3454209